eventlet/wsgi.py

```diff
@@ -392,7 +394,17 @@ def _read_request_line(self):
             return ''
 
         try:
-            return self.rfile.readline(self.server.url_length_limit)
+            if self.server.keepalive_timeout:
```

Comment:
Should we skip this for the first request on a connection?

Reply:
I'd expect keepalive timeout to work on new connection without requests too.